model/src/w3parall.F90

```diff
@@ -1122,7 +1112,9 @@ SUBROUTINE SET_UP_NSEAL_NSEALM(NSEALout, NSEALMout)
        IF (GTYPE .eq. UNGTYPE) THEN
          NSEALout  = PDLIB_NSEAL
          NSEALMout = PDLIB_NSEALM
-         WRITE(*,*) 'PDLIB_NSEAL=', PDLIB_NSEAL, ' PDLIB_NSEALM=', PDLIB_NSEALM
+#endif
+!!/PDLIB         WRITE(*,*) 'PDLIB_NSEAL=', PDLIB_NSEAL, ' PDLIB_NSEALM=', PDLIB_NSEALM
```

Comment:
delete commended out code, also the added #endif and #ifdef W3_PDLIB statements are not needed.